checkpointctl.go

```diff
@@ -133,6 +140,10 @@ func inspect(cmd *cobra.Command, args []string) error {
 		files = true
 	}
 
+	if psTree && psTreeCmd {
+		return fmt.Errorf("cannot use --ps-tree with --ps-tree-cmd")
```

Comment:
Is this a problem? We should silently enable `psTree` when `psTreeCmd = true` anyway